src/clj_momo/lib/es/conn.clj

```diff
@@ -15,6 +15,12 @@
    :content-type :json
    :throw-exceptions false})
 
+(defn make-default-ops [{:keys [_source]}]
+  (cond-> default-opts
+    (seq? _source)
+    (assoc :query-params
+           {:_source (clojure.string/join "," _source)})))
```

Comment:
`:query-params` will never be set in `default-opts`?
You could replace this part with:
```clojure
(update :query-params 
        #(assoc % :_source (clojure.string/join "," _source)))
```